tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,31 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
```

Comment:
This doesn't run at all atm, b/c there is no `it()` in here.
Please test locally by pointing your sdk against staging first.

Reply:
Fixed and test locally.